startapp.sh

```diff
@@ -2,4 +2,9 @@
 
 # Calls the symlink under /bin
 #HOME="/tmp" 
-ilastik
+if [ ! -e /bin/run_ilastik ]; then
```

Comment:
Maybe we should call this something other than run_elastik? The original command that we symlinked also was run_ilastik.sh and it might be confuse maintainers in the future. Maybe start_ilastik?

Reply:
You are right. What about `ilastik_with_args`?

Reply:
Works. BTW, another approach could be to not symlink run_ilastik.sh -> ilastik, but rather create a bash script /bin/ilastik which accepts arguments, if user passes.

Reply:
For me run_ilastik.sh is already a bash script which accepts arguments. What I do not know is how to pass arguments to the docker file `start_app.sh`. Were you thinking about environment variables?

Reply:
Possibly. But still complicated as everything is triggered by the "built-in" init script.

Reply:
So we go for the `ilastik_with_args`?

Reply:
Lets do that. We also need to ensure the name in the wrapper matches.